### PR TITLE
Python dependency bumps

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: requirements/prod.in [hunk 1]
```diff
@@ -9,19 +8,18 @@ django-cache-url==3.4.5
 django-cors-headers==4.3.1
 django-extensions==3.2.3
 django-mozilla-product-details==1.0.3
-django-picklefield==3.1
```
Comment (stevejalim) [inline requirements/prod.in:12]: Long unneeded?

File: basket/news/tests/test_models.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def test_get_tx_message_ids(self):
 
     @override_settings(BRAZE_MESSAGE_ID_MAP={"jeff-dowd": "the-dude"})
     def test_get_tx_message_ids_with_map(self):
-        assert ["the-dude", "jeff-dowd"] == models.BrazeTxEmailMessage.objects.get_tx_message_ids()
+        assert sorted(["the-dude", "jeff-dowd"]) == sorted(models.BrazeTxEmailMessage.objects.get_tx_message_ids())
```
Comment (stevejalim) [inline basket/news/tests/test_models.py:93]: Test stabilization, yeap?

**Review by stevejalim — APPROVED** [review]
r+wc

**Review by pmac — APPROVED** [review]
➕

File: requirements/prod.in (hunk 1 above)
Comment (robhudson) [reply requirements/prod.in:12]: yeah, didn't see any use of it anywhere.

**Pull request merged**